crates/air/src/args.rs

```diff
@@ -20,15 +20,22 @@ pub(crate) enum Command {
     /// Start a language server
     Lsp(LspCommand),
 
-    /// Format a file
+    /// Format a set of files or directories
     Format(FormatCommand),
 }
 
 #[derive(Clone, Debug, Parser)]
 pub(crate) struct LspCommand {}
 
 #[derive(Clone, Debug, Parser)]
+#[command(arg_required_else_help(true))]
```

Comment:
I really don't like "accidental" formatting since formatting is destructive. It needs to be intentional from the user. So with this `air format` will display the help page for format. Use `air format .` to format the current directory.

I think we should use this for all subcommands tbh